### PR TITLE
inform6 6.41-r2

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  Inform 6.41 for Unix (22nd July 2022)
  A specified path is longer than 127 characters.
  make: *** [Makefile:210: std-demos/advent.z5] Error 1
```

**Comment by p-linnane** [comment]
@DavidGriffith Sorry for the ping, but I see the latest few releases have failed to compile on Linux for us. Can you weigh in here if you have time?

File: Formula/inform6.rb [hunk 1]
```diff
@@ -22,6 +22,11 @@ class Inform6 < Formula
   end
 
   def install
+    if OS.mac?
+      inreplace "Makefile", "-DUNIX", "-DMACOS"
+    else
+      inreplace "Makefile", "-DUNIX", "-DLINUX"
+    end
```
Comment (chenrui333) [inline Formula/inform6.rb:29]: We might want to send this patch into `https://gitlab.com/DavidGriffith/inform6unix`

Comment (chenrui333) [reply Formula/inform6.rb:29]: Submit a patch into the upstream, https://gitlab.com/DavidGriffith/inform6unix/-/merge_requests/24

**Comment by DavidGriffith** [comment]
> ~@DavidGriffith Sorry for the ping, but I see the latest few releases have failed to compile on Linux for us. Can you weigh in here if you have time?~
> 
> EDIT: Apologies, did some additional digging and figured out the solution.

I'm a bit unclear on what's going on here, particularly with regard to the pull-request you made at https://gitlab.com/DavidGriffith/inform6unix/-/merge_requests/24.  Is that no longer required?

**Comment by p-linnane** [comment]
The issue we were experiencing was due to UNIX being the default when compiling. I wrote a workaround for the Homebrew formula, but @chenrui333 opened that PR as a fix to make the Makefile more portable in general. If that change is made upstream we won't need a workaround moving forward.

**Comment by chenrui333** [comment]
> > ~@DavidGriffith Sorry for the ping, but I see the latest few releases have failed to compile on Linux for us. Can you weigh in here if you have time?~
> > EDIT: Apologies, did some additional digging and figured out the solution.
> 
> I'm a bit unclear on what's going on here, particularly with regard to the pull-request you made at https://gitlab.com/DavidGriffith/inform6unix/-/merge_requests/24. Is that no longer required?

The PR is trying to address the build issue that we have run into.

**Comment by chenrui333** [comment]
@DavidGriffith does it make sense?

**Review by chenrui333 — APPROVED** [review]

**Comment by DavidGriffith** [comment]
> @DavidGriffith does it make sense?

It does.  I was called away from this for a few days.  I wanted to go over the patch vary carefully to make sure there are no unaccounted side-effects.  While doing this, I noticed that the Inform6 upstream added a specific `-DLINUX` define which is a superset of `-DUNIX`.  That gave me concern and I want to give it a good check-over on *BSD.

**Comment by DavidGriffith** [comment]
I finished checking over things and have committed your patch to the stable and dev branches.  Thanks for bringing this problem to my attention.

**Comment by chenrui333** [comment]
np, thanks for the merge!

**Comment by p-linnane** [comment]
Thank you @DavidGriffith!

**Pull request closed without merging**